CHANGELOG.md

```diff
@@ -10,6 +11,7 @@
     * Hazelcast 5.3.0
     * Swagger 2.2.10
     * Liquibase 4.21.1
+    * cxf 3.5.6
```

Comment:
Siehe Zeile 6/7. bereits vorhanden.

Insgesamt sind die meisten Änderungen hier auch bereits im 1.34 drin, hier hatte ich(?) vergessen das in 1.34 zu schieben und master neu zu erstellen

Reply:
Ich hab mal eine 1.34 erstellt und das alles dahin verschoben.